WPMedia/fusion-news-theme-blocks
# Hotfix revert pen 814

## Description
revert PEN-814 work that landed on stable branch before planed

## Jira Ticket
- [PEN-1462](https://arcpublishing.atlassian.net/browse/PEN-1462)

## Acceptance Criteria
The changes related to 

PEN-814: Display label on promo images for Video and Gallery type objectsReady for Deployment (for XL, L, M and S promo blocks) should not be in stable (what we called 1.3.0 when we released to customers) – these icons should not appear on client production.

These changes should continue to be available in beta (what we called 1.4.0) 

## Test Steps
- check the home page or create a new page looking for this blocks
  - top-table-list-block
  - extra-large-promo-block
  - large-promo-block
  - medium-promo-block
  - small-promo-block

neither block must show the promo label (the label showing the text 'video' or 'gallery' and an icon)

## Effect Of Changes
### Before
![Screen Shot 2020-10-27 at 11 24 13 AM](https://user-images.githubusercontent.com/9757/97369474-c89a2c80-188b-11eb-97c5-8b75104bf680.png)

### After
<img width="668" alt=" 2020-10-27-19 11 56" src="https://user-images.githubusercontent.com/9757/97369452-b8824d00-188b-11eb-95c9-107f68890fa1.png">


## Dependencies or Side Effects
- none

## Review Checklist
please explain why (so that we can fix it whenever it gets refactored).
